controls/radpanelbar/features/resizeable_items.md

```diff
@@ -0,0 +1,55 @@
+---
+title: PanelBar Resizeable Items
+page_title: Resize panel bar items
+description: This article provides information on how to resize the panel bar items
+slug: radpanelbar-features-resize-items
+tags: resize, items, panelbar resize item
+published: True
+position: 0
+---
+
+# Resize RadPanelBar Items
+
+By default the __RadPanelBar__ items cannot be resized dynamically. To enable dynamic resizing on the items you can use the __IsResizingEnabled__ property of the __RadPanelBar__ control.
+
+## Enable Items Resizing
+
+The items resizing is enabled by setting the __IsResizingEnabled__ property to __True__.
+
+#### __[XAML] Example 1: Enable items resizing
+{{region radpanelbar-features-resize-items_0}}
+	<telerik:RadPanelBar IsResizingEnabled="True"/>
+{{endregion}}
+
+The __IsResizingEnabled__ property can be set in code as well.
+
+#### __[C#] Example 2: Enable items resizing in code
+{{region radpanelbar-features-resize-items_1}}
+	radPanelBar.IsResizingEnabled = true;
+{{endregion}}
+
+#### __[VB.NET] Example 2: Enable items resizing in code
+{{region radpanelbar-features-resize-items_2}}
+	radPanelBar.IsResizingEnabled = True
+{{endregion}}
+
+## Control Items Expanded Length
+
+The expanded length of the __RadPanelBarItem__ element can be controlled via the __ExpandedLength__, __ExpandedMinLength__, and __ExpandedMaxLength__ properties. These properties are applied only when the __IsResizingEnabled__ property is set to __True__.
+
+* __ExpandedLength__: Gets or sets the expanded length of the panel bar item.
```

Comment:
Listing properties with their API reference description is not a good practice. We've been using it a lot in the past, but it is not considered a good way of writing this type of documentation, so I recommend you to avoid it.  You can easily get this information from Visual Studio (F12) or the API Reference documentation. In the help documentation it would be better to describe what the properties do with examples and plain text. For example I use the following structure:

"[Description what is expanded length. To be honest, I don't understand what does that mean.]
The expanded length of the items is controlled via the ExpandedLength property of RadPanelBar.

To control the minimum and maximum expanded length, set the ExpandedMinLength and ExpandedMaxLenght properties.

The expanded length values will be applied only if IsResizingEnabled is True.

[code]
[kartinka]"